nuget/Auth0.OidcClient.iOS.nuspec

```diff
@@ -3,15 +3,29 @@
 <package>
   <metadata>
     <id>Auth0.OidcClient.iOS</id>
-    <version>2.4.3</version>
+    <version>3.0.0</version>
     <authors>Auth0</authors>
     <owners>Auth0</owners>
-    <licenseUrl>https://opensource.org/licenses/Apache-2.0</licenseUrl>
+    <license type="expression">Apache-2.0</license>
     <projectUrl>https://github.com/auth0/auth0-oidc-client-net</projectUrl>
     <iconUrl>http://secure.gravatar.com/avatar/805765c256ff8617fcad483b5476faf2</iconUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Auth0 OIDC Client for Xamarin iOS apps</description>
     <releaseNotes>
+      Version 3.0.0
+      - Breaking changes! Please visit our migration guide via a link on
```

Comment:
Is the migration guide in the README or can you link directly? Maybe once that other PR is merged?

Reply:
I thought about it but my main concern is that we may move it in the near future.  I'd personally rather of had it in the wiki than in the repo but I couldn't put it out for PR review there.

Reply:
Additionally NuGet packages do not render links in the change log.